common/jenkins-agents/terraform/docker/Dockerfile.ubi8

```diff
@@ -35,7 +34,7 @@ COPY python_requirements /tmp/requirements.txt
 RUN set -x \
     && dnf -y install $INSTALL_PKGS
 
-RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" \
+RUN curl "https://bootstrap.pypa.io/pip/3.6/get-pip.py" -o "get-pip.py" \
```

Comment:
Hm, will this break again, if `yum-utils` will start using a python version above 3.6?